src/components/PromptForm/DesktopPromptForm.tsx

```diff
@@ -291,19 +304,23 @@ function DesktopPromptForm({
     // Otherwise, let the default paste handling happen
   };
 
+  const activeBorder = useColorModeValue("green.100", "green.600");
+
   return (
     <Flex dir="column" w="100%" h="100%">
       <Card flex={1} my={3} mx={1}>
         <chakra.form onSubmit={handlePromptSubmit} h="100%">
-          <CardBody h="100%" px={6} py={4}>
+          <CardBody
+            h="100%"
+            px={6}
+            py={4}
+            transition="background-color 0.2 ease"
+            _hover={{ borderColor: activeBorder }}
+            borderColor={isDragActive ? activeBorder : "inherit"}
```

Comment:
```suggestion
            border={"2px solid"}
            borderColor={isDragActive ? activeBorder : "inherit"}
```

The reason is there is no border here to start with :)

![image](https://github.com/user-attachments/assets/e6fdf5aa-7765-4286-ad00-23c061ab121d)